### PR TITLE
Nsg multiple sourceip and rocky image plan

### PR DESCRIPTION
**Pull request by hmeiland**
update api version and change prefix to prefixes for nsg
and add rocky plan

### CODE REVIEW

File: pyazhpc/arm.py [hunk 1]
```diff
@@ -689,7 +689,7 @@ def _add_vm(self, cfg, r, vnet_in_deployment):
                             "protocol": "Tcp",
                             "sourcePortRange": "*",
                             "destinationPortRange": "3389",
-                            "sourceAddressPrefix": "*",
+                            "sourceAddressPrefixes": "*",
```
Comment (edwardsp) [inline pyazhpc/arm.py:692]: This is type "string[]" and so needs a list of strings.

**Review by edwardsp — COMMENTED** [review]
I'm not sure how you are using this as there is no example with multiple source address prefixes - how are they being assigned?  Also, throughout you are assigning a string value, which is valid for `sourceAddressPrefix`, but `sourceAddressPrefixes` takes a string list.

File: pyazhpc/arm.py (hunk 1 above)
Comment (hmeiland) [reply pyazhpc/arm.py]: updated

**Comment by hmeiland** [comment]
adding vnc nsg definition

**Review by edwardsp — APPROVED** [review]
looks good

**Pull request merged**